docs/units/equivalencies.rst

```diff
@@ -111,8 +111,7 @@ energy can be converted.
 These equivalencies even work with non-base units::
 
   >>> # Inches to calories
-  >>> from astropy.units import imperial
-  >>> imperial.inch.to(imperial.Cal, equivalencies=u.spectral())  # doctest: +FLOAT_CMP
+  >>> u.imperial.inch.to(u.imperial.Cal, equivalencies=u.spectral())  # doctest: +FLOAT_CMP
```

Comment:
I reverted this change, leaving the `from astropy.units import imperial` in the narrative docs.